src/components/RouteCalendar/RouteCalendar.jsx

```diff
@@ -42,20 +43,25 @@ const RouteCalendar = () => {
   } = useDisclosure();
 
   useEffect(() => {
-    const fetchAllRoutes = async () => {
-      const routesFromDB = await getAllRoutes();
-      const eventsList = routesFromDB.map(route => ({
-        id: route.id,
-        title: route.name,
-        start: new Date(route.date).toISOString().replace(/T.*$/, ''),
-        allDay: true,
-      }));
+    const fetchAllRoutesAndDrivers = async () => {
+      const [routesFromDB, driversFromDB] = await Promise.all([getAllRoutes(), getDrivers()]);
+      const eventsList = routesFromDB.map(route => {
+        const { id, name, date } = route;
```

Comment:
You can directly destructure these attributes in the parameter line for this arrow function, where instead of
`route => { ... }`
we can use
`({ id, name, date }) => { ... }`

Additionally you can also then change the function to remove the `return` keyword by directly returning the object, as following:
`(params) => ({ object_with_attributes })`